### PR TITLE
Update CMakeLists.txt for rpm be usable

### PR DESCRIPTION
**Pull request by bayaro**
+ ignore for some Centos system dirs
+ CPACK_RPM_PACKAGE_RELEASE = CPACK_PACKAGE_RELEASE

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -596,15 +596,19 @@ set(CPACK_PACKAGING_INSTALL_PREFIX "/opt/td-agent-bit")
 set(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}-${CPACK_PACKAGE_RELEASE}.${CMAKE_SYSTEM_PROCESSOR}")
 set(CPACK_RPM_PACKAGE_GROUP "System Environment/Daemons")
 set(CPACK_RPM_PACKAGE_LICENSE "Apache v2.0")
+set(CPACK_RPM_PACKAGE_RELEASE ${CPACK_PACKAGE_RELEASE})
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/cpack/description")
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Fast data collector for Linux")
 set(CPACK_RPM_SPEC_MORE_DEFINE "%define ignore \#")
 set(CPACK_RPM_USER_FILELIST
+        "%ignore /etctd-agent-bit" # kludge to remove badly configured dir
```
Comment (edsiper) [inline CMakeLists.txt]: do you know where is the root cause of that problematic path composition ?, I would prefer to fix the root cause than a workaround.

**Comment by edsiper** [comment]
root path problem fixed by d4a3300e. 

please update PR to add pending changes only..

**Comment by edsiper** [comment]
thanks

**Comment by bayaro** [comment]
> thanks

thank you

**Pull request merged**